app/assets/stylesheets/common/components/user-card.scss

```diff
@@ -1,5 +1,16 @@
-@use "sass:math";
+$usercard-backdrop-avatar: dark-light-choose(
+  blur(30px),
+  blur(30px) brightness(0.7)
+);
 
+$usercard-backdrop: dark-light-choose(none, brightness(0.7));
```

Comment:
The goal here is to have only a backdrop for light color schemes?

Reply:
The other way around:
https://github.com/discourse/discourse/blob/bc5cad217d2abe6861b2540cae33a78da82c66f7/app/assets/stylesheets/common/foundation/variables.scss#L187-L193
I assume we prefer to use a dimmer color for dark mode.